program/convert-string-to-pathcase/convert-string-to-pathcase.java

```diff
@@ -0,0 +1,21 @@
+class replaceSpace
+{
+	static String replaceSpace(String str)
+	{
+		String s = "";
+		for (int i = 0; i < str.length(); ++i) {
+			if (str.charAt(i) == ' ')
+				s += '/';
+			else
+				s += str.charAt(i);
+		}
+		return s;
+	}
+	public static void main(String []args)
+	{
+	    Scanner sc= new Scanner(System.in);
+        System.out.print("Enter a string: ");  

```

Comment:
Copied and pasted code onto online editor, and it got stuck with here
![image](https://user-images.githubusercontent.com/35031228/195214349-5e871f05-450a-4d24-95f0-bcb2c62c83f0.png)